scripts/staging/sobol.dml

```diff
@@ -0,0 +1,1084 @@
+      910     13      2816    1 3 3 7 3 1 61 31 41 533 2025 4067 6963
+      911     13      2821    1 1 5 7 17 27 81 79 107 205 29 97 4883
+      912     13      2831    1 1 1 5 19 49 91 201 283 949 651 3819 5073
+      913     13      2834    1 1 7 9 11 13 73 197 37 219 1931 3369 6017
+      914     13      2839    1 1 7 15 11 7 75 205 7 819 399 661 6487
+      915     13      2845    1 3 3 3 27 37 95 41 307 165 1077 3485 563
+      916     13      2852    1 3 5 3 21 49 57 179 109 627 1789 431 2941
+      917     13      2856    1 1 7 5 11 19 43 137 149 679 1543 245 1381
+      918     13      2861    1 3 5 5 15 3 69 81 135 159 1363 3401 6355
+      919     13      2873    1 3 5 1 9 61 49 53 319 25 1647 1297 615
+      920     13      2874    1 3 5 11 31 43 9 101 71 919 335 3147 5823
+      921     13      2888    1 3 1 1 15 5 29 109 511 945 867 3677 6915
+      922     13      2893    1 3 3 15 17 49 91 111 215 29 1879 97 2505
+      923     13      2894    1 3 1 13 19 61 11 111 163 777 533 1113 5339
+      924     13      2902    1 1 7 9 17 55 117 91 455 289 557 913 4455
+      925     13      2917    1 3 1 7 25 19 123 37 1 277 717 2965 4469
+      926     13      2921    1 3 7 3 19 23 87 235 209 457 2041 2893 1805
+      927     13      2922    1 3 3 5 5 43 23 61 351 791 59 2009 2909
+      928     13      2929    1 1 3 7 5 1 27 231 385 257 1261 2701 1807
+      929     13      2935    1 3 1 1 27 19 87 253 131 685 1743 3983 2651
+      930     13      2946    1 3 7 11 21 17 11 81 191 641 1821 3005 7251
+      931     13      2951    1 3 3 5 15 31 41 213 55 931 1953 49 6037
+      932     13      2957    1 1 7 15 7 27 65 223 113 79 1875 911 5445
+      933     13      2960    1 3 7 7 23 55 51 167 495 25 1585 3447 799
+      934     13      2966    1 1 3 7 27 15 95 193 337 415 975 3085 967
+      935     13      2972    1 1 7 15 19 7 93 41 433 551 401 3169 3971
+      936     13      2976    1 1 7 11 13 15 53 69 433 59 1117 3359 6231
+      937     13      2979    1 1 7 3 23 5 115 201 225 109 1903 3897 6265
+      938     13      2985    1 1 1 11 17 1 39 143 361 659 1105 23 4923
+      939     13      3000    1 1 1 9 27 57 85 227 261 119 1881 3965 6999
+      940     13      3003    1 3 7 7 15 7 107 17 315 49 1591 905 7789
+      941     13      3013    1 3 1 7 29 3 47 237 157 769 839 3199 3195
+      942     13      3018    1 1 3 15 25 39 63 15 111 857 881 1505 7671
+      943     13      3020    1 1 7 1 3 35 41 215 99 895 1025 1483 4707
+      944     13      3025    1 3 5 1 1 31 25 247 113 841 397 1825 6969
+      945     13      3042    1 1 3 5 19 41 49 243 225 973 241 175 1041
+      946     13      3047    1 1 1 7 15 15 105 141 83 75 1675 3523 5219
+      947     13      3048    1 1 7 5 13 27 47 199 445 841 959 1157 2209
+      948     13      3051    1 3 5 15 23 31 31 81 85 33 785 2639 7799
+      949     13      3054    1 1 5 13 21 3 47 99 235 943 1731 2467 7891
+      950
```

Comment:
Hi @mboehm7 , this line uses `bitwise_xor` operation, can I implement a `bitwise_xor`. If yes, any preference as to which components should I change.s